metrics/poseval/poseval.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 The HuggingFace Evaluate Authors.
```

Comment:
A bit unrelated, but should we just massively update all of our code from 2020 to 2022? :joy: